src/eos_editor.cpp

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Ended up not really using this. It wasn't working 100%, probably needs a few more things...oh well. Something we can explore later.